src/components/CreatePostPrompt/CreatePostPrompt.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Card, Form, Button } from 'react-bootstrap';
+import axios from 'axios';
+import { useCookies } from 'react-cookie';
+import { updateUserPosts } from '../../redux/userSlice';
+const CreatePostPrompt = ({ setNewPost }) => {
+    const [cookies] = useCookies(['authToken']);
+    const TOKEN = cookies.authToken;
+    const [content, setContent] = useState('');
+    const [image, setImage] = useState('');
+    const dispatch = useDispatch();
+    function createPostHandler(e) {
+        e.preventDefault();
+        const postData = new FormData();
+        if (content) postData.append('content', content);
+        else {
+            return;
+        }
+        if (image) {
+            postData.append('image', image, image.name);
+        }
+        const headers = {
+            headers: {
+                Authorization: `Token ${TOKEN}`,
+            },
+        };
+        axios
+            .post('http://localhost:8000/api/post/', postData, headers)
```

Comment:
please keep the base paths in environment file. 
`.env`

Reply:
please create a separate js file/package for api integration. and each axios code should be in a separate function. and use those functions in your components.

Reply:
fixed